### PR TITLE
Hindi Language added

### PR DESCRIPTION
**Pull request by evalsocket**
add Hindi Language.

Fixes #[replace brackets with the issue number that your pull request addresses].

Changes proposed in this pull request:

[list out summary of changes here]
[list out summary of changes here]
[list out summary of changes here]
[etc]
Note: pull requests without proper descriptions may simply be closed without further discussion. We appreciate your contributions, but need to know what you are offering in clearly described format. Thanks! (you can delete this text)

cc @HospitalRun/core-maintainers

### CODE REVIEW

File: app/locales/ru/translations.js [hunk 1]
```diff
@@ -883,7 +884,9 @@ export default {
     'es-co': 'Испанский (колумбийский)',
     'pt-br': 'Португальский (бразильский)',
     tr: 'турецкий',
-    ur: 'урду'
+    ur: 'урду',
+    hindi:'хинди'
```
Comment (Kasahs) [inline app/locales/ru/translations.js]: leave space after colon to void style-lint error.

Comment (Kasahs) [inline app/locales/ru/translations.js]: add space after colon to avoid style-lint issues.

File: app/locales/ru/translations.js [hunk 2]
```diff
@@ -883,7 +884,9 @@ export default {
...
+
```
Comment (Kasahs) [inline app/locales/ru/translations.js]: remove trailing spaces on this line to avoid style-lint issues

**Comment by tangollama** [comment]
These changes were merged in another PR. Closing.

**Pull request closed without merging**